src/scripts/components/component.es6

```diff
@@ -96,7 +96,7 @@ module.exports = class Component {
 	destroy() {
 		this.detachListeners()
 
-		for (let prop in this) {
+		for (const prop in this) {
```

Comment:
Čte se mi to právě úplně v pohodě. Ta reference by se uvnitř bloku `{ ... }` ideálně neměla měnit, což je jasné z toho `const`. `let` je zavádějící; naznačuje, že je nějaký záměr tu referenci měnit.